client/components/pickup/PickupCarousel.tsx

```diff
@@ -47,9 +46,43 @@ const PickupCarousel = () => {
 				}}
 			>
 				{slides.map((slide, index) => (
-					<div key={index}>
-						<img src={slide.url} alt={slide.url} />
-					</div>
+					<Box key={index} minWidth={"100%"} style={{ position: "relative" }}>
+						<Image src={slide.url} alt={slide.url} />
+						<div
+							style={{
+								position: "absolute",
+								bottom: isMobile
+									? index === 0
+										? "46%"
+										: "53%"
+									: index === 0
+									? "48%"
+									: "48.5%", // Adjust the positioning as needed
+								left: isMobile
+									? index === 0
+										? "56%"
+										: "30.5%"
+									: index === 0
+									? "71%"
+									: "21%", // Adjust the positioning as needed
+								color: "black",
+							}}
+						>
+							<Box w="70px" alignContent={"center"}>
+								{" "}
+								{index === 0 ? (
+									<Text fontSize={["13px", "13px", "20px"]} as={"b"}>
+										TEST
```

Comment:
is there supposed to be proper text here?

Reply:
This is placeholder for a number that should be calculated and inserted in another branch